### PR TITLE
Added Interactivity

### PR DESCRIPTION
**Pull request by BryteLitty**
In this project, I made the following updates:
- added event listener for checkboxes to mark items as completed on change
- added event listener for clear all button to delete completed items

### CODE REVIEW

File: modules/interactivity.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  clearBtn.addEventListener('click', () => {
+    const newItems = items.filter((item) => item.completed === false);
+    newItems.forEach((todo, idx) => {
+      todo.index = idx + 1;
+    });
+    localStorage.setItem('TodoList', JSON.stringify(newItems));
+    window.location.reload();
+  });
+};
```
Comment (levy002) [inline modules/interactivity.js]: - [ ] Kindly make sure when the user clicks on the `Clear all completed` button, all checked tasks must be deleted but at this moment your app is deleting everything when this button is clicked. :wink: 

   - ### YOUR APP BEHAVIOUR
    
     chrome-capture-2022-10-12 (9)

File: modules/interactivity.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+const itemCheck = () => {
+  const checkboxes = document.querySelectorAll('.checkbox');
+
+  checkboxes.forEach((checkbox, index) => {
+    checkbox.addEventListener('change', (e) => {
+      if (items[index]) {
+        items[index].completed = e.target.checked;
+        localStorage.setItem('TodoList', JSON.stringify(items));
+      }
```
Comment (levy002) [inline modules/interactivity.js]: - [ ] Kindly make sure when the task is checked we only change the `completed` status to `true` or `false` but now when the user clicks the task `checkbox` in your app it changes the task description to static word `on`. Kindly fix that.

    - ### YOUR APP BEHAVIOUR
     chrome-capture-2022-10-12 (9)

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @BryteLitty  , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Santiago220991 — APPROVED** [review]
Hi @BryteLitty, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**